corehq/doctypemigrations/continuous_migrate.py

```diff
@@ -25,6 +26,7 @@ def copy_docs(source_db, target_db, doc_ids):
         return
     with IterDB(target_db, new_edits=False) as iter_db:
         for doc in iter_docs(source_db, doc_ids, attachments=True):
+            doc = _insert_attachments(source_db, doc)
```

Comment:
@dannyroberts I'm not sure why this works. 

`iter_docs` uses dimagi-utils and POSTs a query to "_all_docs" with "attachments=True" as a query param. `_insert_attachments` uses couchdbkit, which uses restkit, which sends a GET request with "attachments=True" as a query param. 

Why does Couch return the attachments the second time but not the first time? ... And if it's that Couch doesn't include attachments in bulk/list queries, is there a reason we call `iter_docs` with the "attachments=True" param?